src/Controllers/AuthController.cs

```diff
@@ -40,6 +40,20 @@ public async Task<ActionResult<UserResponse>> Register(CreateUserDto createUserD
         _dbContext.Users.Add(user);
         await _dbContext.SaveChangesAsync();
 
+        // Todo: Extract into class
```

Comment:
This should be an event, when a user is created an account should be created like:
`UserCreationEvent -> CheckForDefaultAcount`.